docs/pages/versions/unversioned/sdk/sqlite.md

```diff
@@ -89,3 +89,17 @@ A `Transaction` object is passed in as a parameter to the `callback` parameter f
 
 -   **rows._array (_number_)** -- The actual array of rows returned by the query. Can be used directly instead of getting rows through `rows.item()`.
 
+## Execution outside a transaction
+
+  > Please note that you should use this kind of execution only when
+    it is necessary. For instance, when code is no-op within transaction.
```

Comment:
```suggestion
```

I think Markdown would break the line after `when` and we I think don't want it.